src/installer.js

```diff
@@ -1,32 +1,46 @@
 const {exec} = require('@actions/exec')
 const path = require('path')
-const semver = require('semver')
 
-module.exports = {installElixir, installOTP}
+/**
+ * Install Erlang/OTP.
+ *
+ * @param {string} osVersion
+ * @param {string} otpVersion
+ */
+async function installOTP(osVersion, otpVersion) {
+  return await exec(path.join(__dirname, 'install-otp'), [
+    osVersion,
+    otpVersion,
+  ])
+}
 
 /**
  * Install Elixir.
  *
- * @param {string} version
- * @param {string} otpMajor
+ * @param {string} elixirVersion
  */
-async function installElixir(version, otpMajor) {
-  if (process.platform === 'linux') {
-    const otpString = otpMajor ? `-otp-${otpMajor}` : ''
-    await exec(path.join(__dirname, 'install-elixir'), [version, otpString])
-  }
+async function installElixir(elixirVersion) {
+  return await exec(path.join(__dirname, 'install-elixir'), [elixirVersion])
```

Comment:
I think `async` + `return await` is redundant unless you do something like wrap `try .. catch`.

Reply:
Sure. I'll remove it.

Reply:
... though keeping it is not hurtful; I did it, by principle, elsewhere, then must have replicated it here for no special reason.

Reply:
3c2d3a5

Reply:
Is `async` and a single `await` also redundant?

Reply:
Hm... that's a good question. I've always used `async` so that consumers `await` on my function. If `exec` is `async`, though, without `await` the function should just return "instantaneously".

Reply:
```javascript
async function first() {
    await exec()
    console.log('first')
}

async function exec(Cmd) {
    return new Promise((res, rej) => {
        setTimeout(function() {
            res(console.log('do I show up before "first"?'))
        }, 100)
    })
}

first()
```

If your Promise is not synchronous, an `await` is required. 

This is what MDN states "The body of an async function can be thought of as being split by zero or more await expressions. Top-level code, up to and including the first await expression (if there is one), is run synchronously. In this way, an async function without an await expression will **run** synchronously. If there is an await expression inside the function body, however, the async function will always **complete** asynchronously."